bears/general/CustomExceptions.py

```diff
@@ -0,0 +1,11 @@
+class UnmatchedIndentError(Exception):
```

Comment:
Here also, looks like everything below /bears/ is bears and bears only. So this is probably the wrong place for this functionality. I see that coala-bears really only consists of bears, docs, and tests, leaving no proper room for this.

then again, that is what we have coala for, especially /coalib/bearlib/. Especially because it is planned to move all bears (or all bears of certain languages) to their own repositories in the future. So maybe move the functionality in there or keep it in the bear.

A bear defining and raising its own exception is mean though, since it is likely to pass an except clause somewhere and break coala. Can you maybe get around them and stay with stock exceptions?